README.md

```diff
@@ -136,11 +136,18 @@ Define a custom work class:
 # frozen_string_literal: true
 class MyModel < Valkyrie::Resource
   include Valkyrie::Resource::AccessControls
-  attribute :title, Valkyrie::Types::Set       # Sets are unordered
-  attribute :authors, Valkyrie::Types::Array   # Arrays are ordered
+  attribute :title, Valkyrie::Types::Set    # Sets deduplicate values
+  attribute :date, Valkyrie::Types::Array   # Arrays can contain duplicate values
 end
 ```
 
+Attributes are unordered by default.  Adding `ordered: true` to an attribute definition will preserve the
+order of multiple values.
+
+```
+attribute :authors, Valkyrie::Types::Set.meta(ordered: true)
```

Comment:
A `Set` is unordered (by definition at least.) The implementation in Valkyrie isn't inherently unordered, but it seems problematic to call it a Set and then give examples on how to order it.

Reply:
👍 I'll update the doc to use an Array instead.